<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RN08862" name="Command line test" tests="1" failures="1" timestamp="22 Mar 2017 16:59:04 GMT" time="67.694" errors="0">
  <testcase name="testSave" time="0.217" classname="cz.mendelu.swi.eprieskumy.dao.DotaznikDaoTest">
    <failure type="cz.mendelu.pef.swi.eprieskumy.utils.DaoLayerException" message="org.hibernate.PersistentObjectException: detached entity passed to persist: cz.mendelu.pef.swi.eprieskumy.domain.Dotaznik">
      <![CDATA[cz.mendelu.pef.swi.eprieskumy.utils.DaoLayerException: org.hibernate.PersistentObjectException: detached entity passed to persist: cz.mendelu.pef.swi.eprieskumy.domain.Dotaznik
	at cz.mendelu.pef.swi.eprieskumy.dao.DotaznikDaoImpl.save(DotaznikDaoImpl.java:48)
	at cz.mendelu.swi.eprieskumy.dao.DotaznikDaoTest.testSave(DotaznikDaoTest.java:42)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:175)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testSave -->
</testsuite> <!-- Command line test -->
